src/NzbDrone.Core/ImportLists/NYTimes/NYTimesImportApi.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+
+namespace NzbDrone.Core.ImportLists.NYTimes
+{
+    public class NYTimesResponse<T>
+    {
+        public string status { get; set; }
+        public string copyright { get; set; }
+        public int num_results { get; set; }
+        public string last_modified { get; set; }
+        public T results { get; set; }
+    }
+
+    public class NYTimesName
+    {
+        public string list_name { get; set; }
+        public string display_name { get; set; }
+        public string list_name_encoded { get; set; }
+        public string oldest_published_date { get; set; }
+        public string newest_published_date { get; set; }
+        public string updated { get; set; }
+    }
+
+    public class NYTimesList
+    {
+        public string list_name { get; set; }
+        public string display_name { get; set; }
+        public string bestsellers_date { get; set; }
+        public string published_date { get; set; }
+        public int rank { get; set; }
+        public int rank_last_week { get; set; }
+        public int weeks_on_list { get; set; }
+        public int asterisk { get; set; }
+        public int dagger { get; set; }
+        public string amazon_product_url { get; set; }
+        public object[] isbns { get; set; }
+        public List<NYTimesBook> book_details { get; set; }
+        public object[] reviews { get; set; }
+    }
+
+    public class NYTimesBook
+    {
+        public string age_group { get; set; }
+        public string author { get; set; }
+        public string contributor { get; set; }
+        public string contributor_note { get; set; }
+        public string created_date { get; set; }
+        public string description { get; set; }
+        public string price { get; set; }
+        public string primary_isbn13 { get; set; }
+        public string primary_isbn10 { get; set; }
+        public string publisher { get; set; }
+        public int rank { get; set; }
+        public string title { get; set; }
+        public string updated_date { get; set; }
```

Comment:
All these Props should be `TitleCase`. Use `[JsonProperty(PropertyName = "updated_date")]` if needed to match the API props to the Properties

Reply:
Updated props to be TitleCase